meetings/organize.html

```diff
@@ -0,0 +1,97 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+      <meta charset=utf-8">
+      <title>Organizing a distributed meeting</title>
+      <link rel="stylesheet" href="/StyleSheets/generic-base-1.css" type="text/css">
+      <link rel="stylesheet" type="text/css" href="../assets/main.css">
+      <link rel="shortcut icon" href="/Icons/WWW/Literature.gif">
+      <style type="text/css">
+	.issue {
+	outline: solid red;
+	margin-left: 2em;
+	margin-right: 4em;
+	}
+	.issue::before {
+	font-weight: bold;
+	content: "Issue:";
+	}
+
+	.subtopic {
+	font-weight: bold;
+	}
+	.subtopic::after {
+	content: " - ";
+	}
+      </style>
+</head>
+
+<body>
+      <div id="header">
+            <span class="logo"><a href="/"><img src="/Icons/WWW/w3c_home_nb" alt="W3C" height="48"
+                              width="72" /></a></span>
+            <div class="breadcrumb">
+                  <a href="/Member/">Member</a> &rarr;&nbsp;<a href="/Guide/">The Art of Consensus</a> &rarr;&nbsp;
+                  <h1>Organizing a distributed meeting</h1>
+            </div>
+            <p class="baseline">This <strong>Guidebook</strong> is the collected wisdom of the W3C Group Chairs and
+                  other collaborators.</p>
+      </div>
+      <div class="toc">
+            <h4>On This Page&nbsp;&rarr;&nbsp;</h4>
+            <ul>
+                  <li><a href="#timezones">Timezones</a>&nbsp;&bull;&nbsp;</li>
+                  <li><a href="#calendar">Group Calendars</a>&nbsp;&bull;&nbsp;</li>
+            </ul>
+      </div>
+
+      <p>
+        W3C benefits from diverse, inclusive, and worldwide participation. This document provides guidance
+        to those organizing distributed meetings.
+      </p>
+
+      <h2 id='timezones'>Dealing with timezones</h2>
+
+      <blockquote cite='https://www.w3.org/Consortium/Process/#distributed-meeting'>
+            <p>A distributed meeting is one where most of the attendees are expected to participate from remote locations
+               (e.g., by telephone, video conferencing, or IRC).</p>
+            <p>&mdash; <a href='https://www.w3.org/Consortium/Process/#distributed-meeting'>W3C Process Document</a></p>
+      </blockquote>
+      <p>A chair should build consensus about the time slots for distributed meetings and should
+	 reevaluate that consensus on a regular basis, e.g. every 6 months.</p>
```

Comment:
It might be worth mentioning that good times to re-evaluate may be around both the semi-annual spring and fall shifts in summer time, and upon significant changes in membership or participation.  (I think that was generally the TAG's practice for revisiting our meeting times, although some of the summer time changes were an easy "switch back to what we did six months ago, since it worked".)

Reply:
Great suggestion. Added in https://github.com/w3c/Guide/pull/112/commits/5ae2aa492173c69df15bc6c37b36397a4c7731c6